### PR TITLE
Added Twitter summary and card types, typed and tested

### PR DESCRIPTION
**Pull request by michmich112**
@artiebits
Recommended new version `1.3.2`
Added:
 - twitter card input to select between the following card types as defined on [twitter documentation](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards):
   - summary
   - summary_large_image
   - player
 - twitter parameters for player url, player width and player height as defined on [twitter player card docs](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/player-card)
 - Cypress tests to validate both default (summary_large_image), summary and player card types

Typed and  Tested

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (artiebits) [inline cypress/fixtures/example.json:1]: where do you use this file? can't find it

**Comment by artiebits** [comment]
Hi @michmich112, thanks for your contribution, I appreciate it. I left some comments, please take a look.

File: cypress/fixtures/example.json (hunk 1 above)
Comment (michmich112) [reply cypress/fixtures/example.json:1]: i think was auto generated when i ran cypress. fixing it in next commit

File: dev/components/App.svelte [hunk 2]
```diff
@@ -34,12 +33,21 @@
...
-  const twitter = {
-    site: "@username",
-    title: "Twitter Card Title",
-    description: "Description of Twitter Card",
-    image: "https://www.example.com/images/cover.jpg",
-    imageAlt: "Alt text for the card!",
+  const twitter = () => {
+    const playerPath = "/player" === window.location.pathname;
+    const summaryPath = "/summary" === window.location.pathname;
+    console.log((playerPath && 100) || undefined);
```
[the remark this answers is not in the document]
Comment (michmich112) [reply dev/components/App.svelte:39]: +1

**Comment by michmich112** [comment]
Cheers @artiebits my bad it seemed like a husky pre-commit hook decided to keep on formatting the README.
Should be all fixed now

**Review by artiebits — APPROVED** [review]

**Comment by artiebits** [comment]
Hi @michmich112 thanks a lot for your contribution! I will publish a new version of the npm package shortly

**Pull request merged**